examples/IIM/ex1/input2d

```diff
@@ -121,10 +121,15 @@ IIMethod {
    p_calc_width				   = P_CALC_WIDTH
    wss_calc_width			  = WSS_CALC_WIDTH
    min_ghost_cell_width = 6
-   pressure_jump_fe_family = "L2_LAGRANGE"
-   velocity_jump_fe_family = "L2_LAGRANGE"
-   wss_fe_family = "L2_LAGRANGE"
-   tau_fe_family = "L2_LAGRANGE"
+  velocity_jump_fe_family = "L2_LAGRANGE" //"MONOMIAL" 
```

Comment:
@AminKolah is the intention for the comments to suggest other options? If so, I'd suggest making this a bit more explicit. Otherwise, I'd suggest removing the comments.

Also, would you mind aligning the option settings in the input file?